lore/admins/reboot-zuul.md

```diff
@@ -0,0 +1,83 @@
+---
+name: rebooting zuul
+layout: default
+permalink: /lore/admins/reboot-zuul/
+---
+
+# Rebooting Zuul
+
+## Overview
+
+Periodically, Zuul will need to be restarted, whether to pick up new changes or even routine system maintenance.  To do so, you will first need to connect to the zuul node(s) by using the bastion as a jump host.  Currently, one would need to access the nodepool instance in the same manner in order to kill the current running images.
```

Comment:
Might want to add a link to https://github.com/BonnyCI/projman/issues/102 as a reason why we're futzing with node pool nodes, so that we know if/when we should change the docs if the issue is ever resolved.